addon/ng2/utilities/command-helper.ts

```diff
@@ -0,0 +1,16 @@
+import * as config from '../models/config'
```

Comment:
config is used on ln#4 as a fail safe to load the settings from the config file
`command.project.ngConfig = command.project.ngConfig || config.CliConfig.fromProject();`